data/json/proficiencies/misc.json

```diff
@@ -117,6 +117,13 @@
     "description": "Before the cataclysm, you were a helicopter pilot.  Now, you just hope you can fly again.",
     "can_learn": false
   },
+  {
+    "type": "proficiency",
+    "id": "prof_aircraft_mechanic",
+    "name": { "str": "Power and Aircraft Mechanic" },
+    "description": "The knowledge and skill to maintain pre-cataclysm aircraft.",
```

Comment:
```suggestion
    "description": "Before the Cataclysm, the US Government certified you to repair aircraft.  You're not really sure how that's going to help you survive now.",
```

Reply:
I would generalize that a bit, like `"You've been trained and certified to repair aircraft.  You're not really sure how that could help you survive now."`

Reply:
Went with Erk's suggestion, if we do add NPC interaction to get the proficiency then this text makes more sense.